src/stavenote.ts

```diff
@@ -274,13 +336,14 @@ export class StaveNote extends StemmableNote {
       //
       // We also extend the y for each note by a half notehead because the
       // notehead's origin is centered
-      const topNoteBottomY = topNote.getStave().getYForLine(5 - topKeys[0].line + HALF_NOTEHEAD_HEIGHT);
+      const topNoteBottomY = topNote.getStave()?.getYForLine(5 - topKeys[0].line + HALF_NOTEHEAD_HEIGHT);
```

Comment:
`topNote.getStave()` must be defined, so add an exception check here and remove the `?`.

Reply:
done